src/components/calcite-block/calcite-block.tsx

```diff
@@ -156,7 +165,11 @@ export class CalciteBlock {
     const hasContent = !!Array.from(el.children).some((child) => child.slot !== CONTROL_SLOT_NAME);
 
     return (
-      <Host aria-expanded={collapsible ? (open ? "true" : "false") : null}>
+      <Host
+        aria-expanded={collapsible ? (open ? "true" : "false") : null}
+        aria-busy={this.loading}
```

Comment:
nitpicks: don't use this. add the props where `collapsible` is added.

Reply:
done. 🌮

Reply:
ohnvmd 
will fix